### PR TITLE
Improvement : install plugin options

### PR DESCRIPTION
**Pull request by justinehell**
Add prompt to choose between official or custom plugins via `install` command.
Resolve https://github.com/ditrit/leto-modelizer-plugin-cli/issues/8

### CODE REVIEW

File: dist/leto-modelizer-plugin-cli.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
...
...abs(t.length-e.length)>3)return Math.max(t.length,e.length);const i=[];for(let e=0;e<=t.length;e++)i[e]=[e];for(let t=0;t<=e.length;t++)i[0][t]=t;for(let r=1;r<=e.length;r++)for(let s=1;s<=t.length;s++){let n=1;n=t[s-1]===e[r-1]?0:1,i[s][r]=Math.min(i[s-1][r]+1,i[s][r-1]+1,i[s-1][r-1]+n),s>1&&r>1&&t[s-1]===e[r-2]&&t[s-2]===e[r-1]&&(i[s][r]=Math.min(i[s][r],i[s-2][r-2]+1))}return i[t.length][e.length]}(t,e),n=Math.max(t.length,e.length);(n-i)/n>.4&&(i<s?(s=i,r=[e]):i===s&&r.push(e))})),r.sort(((t,e)=>t.localeCompare(e))),i&&(r=r.map((t=>`--${t}`))),r.length>1?`\n(Did you mean one of ${r.join(", ")}?)`:1===r.length?`\n(Did you mean ${r[0]}?)`:""}}},e={};function i(r){var s=e[r];if(void 0!==s)return s.exports;var n=e[r]={id:r,loaded:!1,exports:{}};return t[r](n,n.exports,i),n.loaded=!0,n.exports}i.nmd=t=>(t.paths=[],t.children||(t.children=[]),t);var r,s,n,o={};r=i(7461).Command,s=i(1307),(n=new r).name("leto-modelizer-plugin-cli").description("CLI install and manage plugins in leto-modelizer").version("0.2.1"),Object.keys(s).forEach((function(t){return s[t].setup(n)})),n.parse();var h=exports;for(var a in o)h[a]=o[a];o.__esModule&&Object.defineProperty(h,"__esModule",{value:!0})})();
```
Comment (Zorin95670) [inline dist/leto-modelizer-plugin-cli.js:2]: Remove this from PR, it's the release job

File: README.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Generate issues report with eslint for sonar.
 To install the client in Leto Modelizer, use this command line:
 
 ```
-npm install --save-dev "git://github.com/ditrit/leto-modelizer-plugin-cli.git#0.1.1"
+npm install --save-dev "git://github.com/ditrit/leto-modelizer-plugin-cli.git#0.2.1"
```
Comment (Zorin95670) [inline README.md:40]: Add explanation on how to write configuration file.

File: dist/leto-modelizer-plugin-cli.js (hunk 1 above)
Comment (justinehell) [reply dist/leto-modelizer-plugin-cli.js:2]: done

File: README.md (hunk 2 above)
Comment (justinehell) [reply README.md:40]: done

**Pull request closed without merging**